web-portal/store/index.js

```diff
@@ -173,6 +173,7 @@ export const mutations = {
 
 export const actions = {
   nuxtServerInit: (ctx, { req }) => {
+    debugger
```

Comment:
Should probably remove this before merging it.

Reply:
done

Reply:
thx